### ISSUE TITLE
ie6 fails due to unimplemented function shlwapi.dll.SHRegOpenUSKeyW

### ISSUE THREAD

**Issue by GoogleCodeExporter** [noahocker/winetricks#329]
```
The ie6 verb adds a native workaround for slwapi.dll but this leads to a crash 
due to unimplemented function shlwapi.dll.SHRegOpenUSKeyW in 
wine-1.6-rc4-122-g104adb7.

The Wine builtin does seem to have this function implemented but when I try to 
changed this back to builtin Wine loads builtin iexplore instead, no matter 
what I do! I haven't the first clue why.
http://source.winehq.org/source/dlls/shlwapi/shlwapi.spec#L737

I tried the shlwapi from my Windows 7 partition (probably IE9) but this 
requires KernelBase.dll and this triggers an unimplemented function crash in 
NTDLL.

I cannot work out how to work around this.
```

Original issue reported on code.google.com by `ImWellCu...@gmail.com` on 12 Jul 2013 at 5:10

**Comment by GoogleCodeExporter** [comment]
```
When does the unimplemented function crash happen - when starting ie6?
```

Original comment by `daniel.r...@gmail.com` on 12 Jul 2013 at 5:17

**Comment by GoogleCodeExporter** [comment]
```
Yup. As soon as.

Just found that IE7 immediately crashed in SHLWAPI.dll.154 too.

I have no idea why WINEDLLOVERRIDES=shlwapi=b has no effect at all!
```

Original comment by `ImWellCu...@gmail.com` on 12 Jul 2013 at 5:31

**Comment by GoogleCodeExporter** [comment]
```
Ah, setting shlwapi to builtin in winecfg works around the IE7 crash. It 
crashes in something else but there you go.
```

Original comment by `ImWellCu...@gmail.com` on 12 Jul 2013 at 5:33

**Comment by GoogleCodeExporter** [comment]
```
IE6 should be removed, honestly. It works poorly and wine can't really use its 
dlls. IE7 should be used instead, for native mshtml/etc.

I can't reproduce this, btw. I ran winetricks -q ie6, then ran iexplore.exe, 
worked fine.
```

Original comment by `austinenglish@gmail.com` on 30 Jul 2013 at 12:12

**Comment by GoogleCodeExporter** [comment]
```
SHRegOpenUSKeyW has been implemented in wine-1.7.27
Also, I tried winetricks -q ie6, it works fine.
Should this issue be closed?
```

Original comment by `litimetal` on 2 Oct 2014 at 12:30

**Comment by GoogleCodeExporter** [comment]
```
Can't reproduce.
```

Original comment by `austinenglish@gmail.com` on 7 Oct 2014 at 4:34
- Changed state: **WorksForMe**
